stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkEventReporter.java

```diff
@@ -217,6 +266,56 @@ InputStream interpretResponseStream(
     boolean fromDiskCache();
   }
 
+  interface InspectorWebSocketRequest extends InspectorRequestCommon {
+  }
+
+  interface InspectorWebSocketResponse extends InspectorResponseCommon {
+    /**
+     * Optional and redundant set of headers from {@link InspectorWebSocketRequest} that are for
+     * some mysterious reason are included here in the response by Chrome.
+     */
+    @Nullable
+    InspectorHeaders requestHeaders();
+  }
+
+  interface InspectorWebSocketFrame {
+    String requestId();
+
+    int OPCODE_CONTINUATION = 0;
+    int OPCODE_TEXT = 1;
+    int OPCODE_BINARY = 2;
+    int OPCODE_CONNECTION_CLOSE = 8;
+    int OPCODE_PING = 9;
+    int OPCODE_PONG = 10;
+
+    int opcode();
+    boolean mask();
```

Comment:
I intentionally left this blank because I don't know what it does in Chrome.  I know what it does in WebSockets, but not in Chrome :(